jsonrpc.go

```diff
@@ -18,20 +18,21 @@ import (
 
 // RPCConfig is the configuration for the API handler
 type RPCConfig struct {
-	WalletTLSEnable    bool
-	WalletTLSKeyFile   string
-	WalletTLSCertFile  string
-	WalletRPCUser      string
-	WalletRPCPassword  string
-	WalletServer       string
-	WalletTimeout      time.Duration
-	WalletCORSDomains  string
-	FactomdTLSEnable   bool
-	FactomdTLSCertFile string
-	FactomdRPCUser     string
-	FactomdRPCPassword string
-	FactomdServer      string
-	FactomdTimeout     time.Duration
+	WalletTLSEnable       bool
+	WalletTLSKeyFile      string
+	WalletTLSCertFile     string
+	WalletRPCUser         string
+	WalletRPCPassword     string
+	WalletServer          string
+	WalletTimeout         time.Duration
+	WalletCORSDomains     string
+	FactomdTLSEnable      bool
+	FactomdTLSCertFile    string
+	FactomdRPCUser        string
+	FactomdRPCPassword    string
+	FactomdServer         string
+	FactomdTimeout        time.Duration
+	WalletWhiteListEnable bool
```

Comment:
whitelist is a single word and should be camel cased Whitelist (like you did in other places actually)